qiita_pet/templates/study_description.html

```diff
@@ -152,6 +152,25 @@
   }
 }
 
+
+function delete_raw_data(raw_data_id) {
+  if (confirm("Are you sure you want to delete raw data " + raw_data_id + "?")) {
```

Comment:
Minor, but I think this could be `'Are you sure you want to delete FASTQ (ID: '+raw_data_id') ?'`.